### DISCUSSION TITLE
Change the background of the nodes with API?

### DISCUSSION THREAD

**Discussion by Hadiasemi** [xyflow/xyflow#3226] [Q&A] [answered]
Hi, I am trying to change the background of each nodes based on API. Is there any resources that you can refer me to or any code snippet will be great. I appreciate your help.

**Comment by bcakmakoglu** [comment, marked as answer]
Styling a node can be done by just passing `node.style`

```ts
  {
    id: "1",
    type: "input",
    data: { label: "Node 1" },
    position: { x: 250, y: 5 },
    style: { backgroundColor: "#2196f3", color: "white", borderColor: "white" }
  }
```

**Reply by Hadiasemi** [reply]
thanks
